lib/proto/service.proto

```diff
@@ -135,38 +137,13 @@ message GetAddressUtxosReplyList {
     repeated GetAddressUtxosReply addressUtxos = 1;
 }
 
```

Comment:
We'll need to come up with a replacement story shortly...   as a short-term solution we could delegate this entirely into the front-end app software.